Index: grpc-web/javascript/net/grpc/web/generator/Makefile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/grpc-web/javascript/net/grpc/web/generator/Makefile b/grpc-web/javascript/net/grpc/web/generator/Makefile
--- a/grpc-web/javascript/net/grpc/web/generator/Makefile	
+++ b/grpc-web/javascript/net/grpc/web/generator/Makefile	
@@ -13,9 +13,9 @@
 # limitations under the License.
 
 CXX ?= g++
-CPPFLAGS += -I/usr/local/include -pthread
+CPPFLAGS += -I/usr/local/include -I/opt/homebrew/include -pthread
 CXXFLAGS += -std=c++11
-LDFLAGS += -L/usr/local/lib -lprotoc -lprotobuf -lpthread -ldl
+LDFLAGS += -L/usr/local/lib -L/opt/homebrew/lib -lprotoc -lprotobuf -lpthread -ldl
 PREFIX ?= /usr/local
 MIN_MACOS_VERSION := 10.7 # Supports OS X Lion
 STATIC ?= yes
